packages/utils/src/event-bus/index.ts

```diff
@@ -23,24 +23,15 @@ export abstract class AbstractEventBusModuleService
   ): Promise<void>
   abstract emit<T>(data: EventBusTypes.EmitData<T>[]): Promise<void>
 
-  public subscribe(
-    eventName: string | symbol,
-    subscriber: EventBusTypes.Subscriber,
-    context?: EventBusTypes.SubscriberContext
-  ): this {
-    if (typeof subscriber !== `function`) {
-      throw new Error("Subscriber must be a function")
-    }
-    /**
-     * If context is provided, we use the subscriberId from it
-     * otherwise we generate a random using a ulid
-     */
-
-    const randId = ulid()
-    const event = eventName.toString()
-
-    const subscriberId = context?.subscriberId ?? `${event}-${randId}`
-
+  protected storeSubscribers({
```

Comment:
suggestion(non-blocking):
```suggestion
  protected addSubscriber({
```

Reply:
I wen for store to avoid confusion between add and subscribe. Just wanted to pass the meaning that we are storing it,  but fine to change it